build_tooling/storage_common.py

```diff
@@ -0,0 +1,57 @@
+from arcticdb import Arctic
+import pandas as pd
+import os
+import numpy as np
+
+LIBRARIES = [
+    # LINUX
+    "linux_cp36",
+    "linux_cp36_compat36",
+    "linux_cp37",
+    "linux_cp38",
+    "linux_cp38_compat38",
+    "linux_cp39",
+    "linux_cp310",
+    "linux_cp311",
+    # WINDOWS
+    "windows_cp37",
+    "windows_cp38",
+    "windows_cp39",
+    "windows_cp310",
+    "windows_cp311",
+]
+
+
+def normalize_lib_name(lib_name):
```

Comment:
I think that there is a problem with the dot in S3, the dot and everything after it are cut off by S3.

In general, I wanted the names to have consistent separators, as it was easier to check what is happening in the S3 bucket during testing

Reply:
All of the library names inside the building have a dot to separate the user/namespace (which corresponds to the storage config database name).

That handling is perhaps incorrectly patched in the external API? Please raise a bug.

Reply:
I have created a bug [here](https://github.com/man-group/ArcticDB/issues/760), but this might only be a UI bug in AWS, as I can read/write to the library without a problem.